nncf/onnx/quantization/quantize.py

```diff
@@ -54,7 +51,8 @@ def quantize_impl(model: onnx.ModelProto,
         preset=preset,
         target_device=target_device,
         number_samples=subset_size,
-        ignored_scopes=convert_ignored_scope_to_list(ignored_scope)
+        ignored_scopes=convert_ignored_scope_to_list(ignored_scope),
+        fast_bias_correction=fast_bias_correction
```

Comment:
What do you think about extending the following test by fast_bias_correction=False parameter https://github.com/openvinotoolkit/nncf/blob/develop/tests/post_training/test_quantize_conformance.py?

Reply:
Done